platform/ios/src/MGLMapView.mm

```diff
@@ -1619,6 +1748,73 @@ - (void)handleTwoFingerDragGesture:(UIPanGestureRecognizer *)twoFingerDrag
         [self notifyGestureDidEndWithDrift:NO];
         [self unrotateIfNeededForGesture];
     }
+
+}
+
+- (MGLMapCamera *)currentCameraWithEstimatedEndPoint:(CGPoint)endPoint panGesture:(UIPanGestureRecognizer *)pan
```

Comment:
`pan` is unused.

A better place for this logic would be a private `-[MGLMapCamera cameraByPanningWithTranslation:inMapView:]` (which would have parameters similar to `-[UIPanGestureRecognizer setTranslation:inView:]`). You’d need to create a new MGLMapCamera_Private.h for these methods.

This way you can also share this and similar methods between iOS and macOS. The downside is that our non-UI code would have some dependency on our UI code, but that won’t introduce any practical problems unless we decide to split the Darwin code into a separate framework down the line.